parser-typechecker/src/Unison/Server/Errors.hs

```diff
@@ -57,16 +59,30 @@ rootBranchError rbe = err500
 
 badNamespace :: String -> String -> ServerError
 badNamespace err namespace = err400
-  { errBody = "Malformed namespace: "
+  { errBody = "Malformed namespace name: "
               <> mungeString namespace
               <> ". "
               <> mungeString err
   }
 
+badDefinition :: String -> String -> ServerError
+badDefinition err definition =
+  err400
+    { errBody =
+        "Malformed definition name: "
+          <> mungeString definition
+          <> ". "
+          <> mungeString err
+    }
+
 noSuchNamespace :: HashQualifiedName -> ServerError
 noSuchNamespace namespace =
   err404 { errBody = "The namespace " <> munge namespace <> " does not exist." }
 
+noSuchDefinition :: HashQualifiedName -> ServerError
+noSuchDefinition definition  =
+  err404 { errBody = "The definition " <> munge definition  <> " does not exist." }
```

Comment:
This isn't used yet, but should be the response if we can't find the definition.